iocore/net/I_NetVConnection.h

```diff
@@ -541,6 +545,31 @@ class NetVConnection : public VConnection
   unsigned int attributes;
   EThread *thread;
 
+  // es - origin_trace associated connections
```

Comment:
yes, pulled from SSLNetVConnection. And put the wiretrace logic into raw_read/write and read/write function that make the code simply and clearly. Just call NetProfileSM::read()/write()/raw_read()/raw_write() and you will get wiretrace feature.